diff -r 6045481e177c sample/main_dec_test.c
--- a/sample/main_dec_test.c	Thu Aug 29 20:58:10 2024 +0800
+++ b/sample/main_dec_test.c	Tue Dec 10 15:51:08 2024 +0700
@@ -28,6 +28,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <strings.h>
+#include <sys/time.h>
 
 #include "BufferAllocatorWrapper.h"
 #include "jpuapi.h"
diff -r 6045481e177c sample/main_enc_test.c
--- a/sample/main_enc_test.c	Thu Aug 29 20:58:10 2024 +0800
+++ b/sample/main_enc_test.c	Tue Dec 10 15:51:08 2024 +0700
@@ -10,6 +10,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <sys/mman.h>
+#include <sys/time.h>
 #include <sys/types.h>
 #include <unistd.h>
 
